templates/android.js

```diff
@@ -1,18 +1,21 @@
 module.exports = platform => [{
   name: () => `${platform}/build.gradle`,
-  content: ({ packageIdentifier }) => `buildscript {
-    ext.safeExtGet = {prop, fallback ->
-        rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
-    }
-    repositories {
-        google()
-        jcenter()
-    }
+  content: ({ packageIdentifier }) => `def safeExtGet(prop, fallback) {
```

Comment:
```suggestion
  content: ({ packageIdentifier }) => `// ${platform}/build.gradle

def safeExtGet(prop, fallback) {
```

(cleaner)

I started this kind of change at a more global level in draft PR #137

Reply:
I'm not sure I understand. Would you then strip out these file headers again before writing them to disk?

Reply:
All I am doing is adding a comment to the beginning of the generated output, to avoid keeping the JavaScript function code and generated Gradle code on the same line in `templates/android.js`. This cleanup is now in draft PR #137, where I explain the twin benefits of this kind of cleanup.

I hope this is clear, please feel free to ask any questions if not.

Reply:
Mmh but would that comment also exist in the generated _files_? Or just here in the source code? In any case, you already mentioned that it will be part of a separate PR.

Reply:
> Mmh but would that comment also exist in the generated _files_?

Yes, and I think this should be a good thing or at least not a bad thing. This is now in PR #137 and #139.